extruct/tool.py

```diff
@@ -6,10 +6,13 @@
 from extruct.jsonld import JsonLdExtractor
 from extruct.rdfa import RDFaExtractor
 from extruct.w3cmicrodata import MicrodataExtractor
+from extruct.opengraph import OpenGraphExtractor
+from extruct.microformat import MicroformatExtractor
 from extruct.xmldom import XmlDomHTMLParser
 
 
-def metadata_from_url(url, microdata=True, jsonld=True, rdfa=True):
+def metadata_from_url(url, microdata=True, jsonld=True, rdfa=True,
+                      microformat=True, opengraph=True):
     resp = requests.get(url, timeout=30)
```

Comment:
@kmike, @cathalgarvey  should we add headers here? Iirc some domains ban requests that do not have one

Reply:
I think it's fine without headers, I'd prefer keeping tool.py as simple as possible.